src/modules/chats/components/chatDetails/index.tsx

```diff
@@ -16,27 +15,13 @@ export const ChatDetails: React.FC<ChatDetailsProps> = ({
   newSocketMessage,
   setChats,
 }) => {
-  const { messages, onSend, account, loadMore, hasMore, page, setMessages } = useChatDetails(
+  const { sortedMessages, onSend, account, loadMore, hasMore, onAvatarClick } = useChatDetails(
     selectedChatId,
     chats,
     setChats,
+    newSocketMessage,
   );
 
-  const sorted = messages?.sort((a, b) => (new Date(a.created_at) > new Date(b.created_at) ? 1 : -1));
-
-  useEffect(() => {
-    if (page === 1) {
-      const messageBody = document.getElementById('chat-list-div');
-      if (messageBody) messageBody.scrollTop = messageBody.scrollHeight - messageBody.clientHeight;
-    }
-  }, [messages]);
-
-  useEffect(() => {
-    if (newSocketMessage && !messages.find(item => item.id === newSocketMessage.id)) {
-      setMessages([...messages, newSocketMessage]);
-    }
-  }, [newSocketMessage]);
-
```

Comment:
For my information:
you have removed this useEffect and put it in messageDeatils, the idea has been moving the person who has sent the most recent message to you to the top of the persons list, how do you handle it?